package.json

```diff
@@ -94,6 +96,7 @@
     "child-process-promise": "^2.2.0",
     "css-loader": "^0.27.3",
     "denodeify": "^1.2.1",
+    "detective-es6": "^1.1.6",
```

Comment:
yes it does. I started out with detective on `/dist` but realized that we can't do that because our bundles bring all the code in, so there aren't actually any `require` calls.